src/patterns/ec2-app/__snapshots__/base.test.ts.snap

```diff
@@ -91,10 +91,6 @@ exports[`the GuEC2App pattern can produce a restricted EC2 app locked to specifi
         "MetricsCollection": [
           {
             "Granularity": "1Minute",
-            "Metrics": [
-              "GroupTotalInstances",
-              "GroupInServiceInstances",
-            ],
           },
```

Comment:
From the docs:

> If you specify `Granularity` and don't specify any metrics, all metrics are enabled.
> – https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-autoscaling-autoscalinggroup-metricscollection.html#cfn-autoscaling-autoscalinggroup-metricscollection-metrics

Reply:
Oh, I did not notice that before! Fair enough, sounds good!